### PR TITLE
Bluewash

### PR DESCRIPTION
**Pull request by mathieucarbou**
This PR might have an impact on all projects depending on that plus eventually some jobs because of the property now being required to be given from elsewhere

### CODE REVIEW

**Comment by akomakom** [comment]
Why do we want to explicitly specify nexus url at build time?

File: pom.xml [hunk 1]
```diff
@@ -102,15 +103,17 @@
...
-    <terracotta-snapshots-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-snapshots</terracotta-snapshots-url>
-    <terracotta-ee-snapshots-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-ee-snapshots</terracotta-ee-snapshots-url>
-    <terracotta-staging-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-staging</terracotta-staging-url>
-    <terracotta-os-releases-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-os-releases</terracotta-os-releases-url>
-    <terracotta-releases-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-releases</terracotta-releases-url>
-    <terracotta-ee-releases-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-ee-releases</terracotta-ee-releases-url>
-    <terracotta-patches-url>https://nexus.terracotta.eur.ad.sag:8443/content/repositories/terracotta-patches</terracotta-patches-url>
-    <terracotta-nexus-staging-url>https://nexus.terracotta.eur.ad.sag:8443/service/local/staging/deploy/maven2</terracotta-nexus-staging-url>
+    <terracotta.nexus.url.url>UNSET</terracotta.nexus.url.url>
```
Comment (akomakom) [inline pom.xml:106]: extra `.url`, FYI

**Review by chrisdennis — COMMENTED** [review]
The copyright changes in here look reasonable. I'm not informed enough to speak to how these parent poms are currently being used so I'm going to let @akomakom and @mathieucarbou decide what is best here.

File: pom.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 <!--
   CHANGELOG (significant changes)
+  8.0: Bluewash
```
Comment (akomakom) [inline pom.xml:5]: I'd also mention requiring nexusURL

**Review by akomakom — APPROVED** [review]

**Pull request merged**